<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lorena - Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Dropbox/Sources/lorena/ui/pixmaps/mameicon.png</normaloff>../../../Dropbox/Sources/lorena/ui/pixmaps/mameicon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabVideo">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="title">
          <string>Size</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QCheckBox" name="fullscreen">
          <property name="geometry">
           <rect>
            <x>17</x>
            <y>33</y>
            <width>102</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Full Screen</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>161</width>
            <height>52</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="scale">
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <zorder>layoutWidget</zorder>
         <zorder>fullscreen</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Frameskip</string>
         </property>
         <widget class="QCheckBox" name="fsautomatic">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>125</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Automatically</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>50</y>
            <width>161</width>
            <height>52</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Effect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fseffect">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Async</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yv12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yvy2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yv12x2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yuy2x2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>111</width>
            <height>52</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Manual</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fsmanual"/>
           </item>
          </layout>
         </widget>
         <zorder>layoutWidget</zorder>
         <zorder>fsautomatic</zorder>
         <zorder>layoutWidget</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHardware">
      <attribute name="title">
       <string>Hardware</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="hwspeed">
         <property name="text">
          <string>Hardware Speed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="prspeed">
         <property name="text">
          <string>Processor Speed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="joystick">
         <property name="text">
          <string>Joystick</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="gameinfo">
         <property name="text">
          <string>Game Information</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPath">
      <attribute name="title">
       <string>File Path</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Mame Binary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="mamepath"/>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="btnMame">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Roms</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="rompath"/>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="btnRom">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Snap</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="snappath"/>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="btnSnap">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Video</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="videopath"/>
       </item>
       <item row="7" column="1">
        <widget class="QToolButton" name="btnVideo">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
